Code/GraphMol/FileParsers/TDTWriter.cpp

```diff
@@ -58,7 +58,7 @@ TDTWriter::TDTWriter(std::ostream *outStream, bool takeOwnership) {
   df_writeNames = true;
 }
 
-TDTWriter::~TDTWriter() throw() {
+TDTWriter::~TDTWriter() {
```

Comment:
I've been meaning to fix this for a while.  Thanks.

Reply:
No problem :)

Reply:
wow is that an old wart... thanks